wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/CertificateDetailsView.swift

```diff
@@ -0,0 +1,108 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+
+struct CertificateDetailsView: View {
+    @Environment(\.dismiss) private var dismiss
+    var certificateDetails: String
+    @State var isMenuPresented = false
+    var isDownloadAndCopyEnabled: Bool = true
+    var performDownload: (() -> Void)?
+    var performCopy: ((String) -> Void)?
+    var body: some View {
+        HStack {
+            Spacer()
+            Text("Certificate Details")
+            Spacer()
+            SwiftUI.Button(action: {
+                dismiss()
+            }, label: {
+                Image(.close)
+            }).padding()
+        }.background(Color.backgroundColor)
+        ScrollView {
+            Text(certificateDetails)
+                .font(Font.sfMonoSmall)
+                .padding()
+                .frame(maxHeight: .infinity)
+        }
+
+        .safeAreaInset(edge: .bottom, spacing: 0) {
+            VStack {
+                if isDownloadAndCopyEnabled {
+                    HStack {
+                        SwiftUI.Button(action: {
+                            performDownload?()
+                        },
+                                       label: {
+                            Image(.download)
+                        }).padding()
+                        SwiftUI.Button(action: {
+                            performDownload?()
+                        },
+                                       label: {
+                            Text("content.message.download".localized).font(UIFont.normalRegularFont.swiftUIfont.bold())
```

Comment:
We're trying to avoid using .localized. Instead we're trying to use L10n.Localizable.Content.Message.Download

Reply:
Done